src/components/Disputes/DisputeEvidences.js

```diff
@@ -133,7 +133,7 @@ const EvidenceContent = React.memo(function EvidenceContent({
               ${textStyle('body2')};
             `}
           >
-            {dayjs(createdAt).format('DD/MM/YY')}
+            {dayjs(toDate(createdAt)).format('DD/MM/YY')}
```

Comment:
Not particularly relevant, but I was looking at the `toDate()` implementation and was quite surprised to see it doing string operations on the returned ISO string.

Maybe `toDate()` can just return `new Date(unixSeconds * 1000)`?

There is also [`dayjs.unix()`](https://day.js.org/docs/en/parse/unix-timestamp)

Reply:
@fabriziovigevani could we change that function? or was written like this for some reason?

Reply:
I don't recall implementing that function. Maybe you added it for the dispute detail PR ?